packages/shared/src/lib/featureManagement.ts

```diff
@@ -24,6 +24,10 @@ const feature = {
   bookmarkOnCard: new Feature('bookmark_on_card', false),
   onboardingVisual: new Feature('onboarding_visual', {
     showCompanies: true,
+    fullBackground: {
+      mobile: cloudinary.onboarding.fullBackground.mobile,
+      desktop: cloudinary.onboarding.fullBackground.desktop,
+    },
```

Comment:
Here we're setting it as a default value, so let's better leave the last value at the moment if this is going to be an experiment

Reply:
As long as they don't get sent from growthbook, it'll be undefined. But I'll set `fullBackground` to false